examples/diff.c

```diff
@@ -66,6 +75,9 @@ static void parse_opts(struct opts *o, int argc, char *argv[]);
 static int color_printer(
 	const git_diff_delta*, const git_diff_hunk*, const git_diff_line*, void*);
 static void diff_print_stats(git_diff *diff, struct opts *o);
+int extract_break_rewrites(int *m, int *n, const char *arg);
+void die(char* message);
```

Comment:
Yeah, I will remove them, also need to remove pair struct above, is there is other problem with code ?